### PR TITLE
Adding Codio-specific changes

### PR DESCRIPTION
**Pull request by edeitrick**

### CODE REVIEW

File: config/environments/development.rb [hunk 1]
```diff
@@ -8,6 +8,8 @@
 
   # Do not eager load code on boot.
   config.eager_load = false
+  
+  config.hosts = nil
```
Comment (armandofox) [inline config/environments/development.rb:12]: i think this is only necessary for Rails >=6, and this assignment uses Rails 4.  i'm OK including it here in anticipation of upgrading the assignment later, as long as we know with confidence that it doesn't break Rails 4.  @edeitrick can you confirm that including this info for Rails 4 apps does not break the app either on Codio or locally?

Comment (edeitrick) [reply config/environments/development.rb:12]: The stack on Codio is rails 5.2.3 and I needed this line to make it work in Codio. At one point I had directions telling the students to add it manually but it seemed cumbersome

**Comment by armandofox** [comment]
>    config.eager_load = false
> +  
> +  config.hosts = nil
> The stack on Codio is rails 5.2.3 and I needed this line to make it work in Codio. At one point I had directions telling the students to add it manually but it seemed cumbersome
> 
> 
not sure i understand tho since the Gemfile supplied with the project says Rails 4.2.9.  again, as long as it doesn't break Rails 4 it's fine to include, but i'd think if you follow the instructions and start by running bundle install, Rails 4 is what will actually be used?

**Comment by edeitrick** [comment]
> config.eager_load = false + + config.hosts = nil The stack on Codio is rails 5.2.3 and I needed this line to make it work in Codio. At one point I had directions telling the students to add it manually but it seemed cumbersome
> not sure i understand tho since the Gemfile supplied with the project says Rails 4.2.9. again, as long as it doesn't break Rails 4 it's fine to include, but i'd think if you follow the instructions and start by running bundle install, Rails 4 is what will actually be used?

I just tried it and even after bundle it's on 5 -- taking a look at the repo it looks like it got bumped in 2019:
image

**Comment by armandofox** [comment]
hmm, i better pull and make sure i'm on the latest version...thanks for the heads up

Armando Fox (he, him, él)
Professor, Computer Science Division • Faculty Advisor, Digital Learning Strategy
UC Berkeley Campus Equity Advisor
390 Soda Hall MC#1776, Berkeley, CA, 94720-1776 • +1.510.642.6820 / http://www.cs.berkeley.edu/~fox <http://www.cs.berkeley.edu/~fox> • PGP/GPG key ID: 0x9AD0E747
-----
Washing your hands after being around others is a personal choice. 
Wearing a mask around others is a choice to be responsible and considerate.

**Pull request merged**